deep-security-agent/README.md

```diff
@@ -46,4 +46,8 @@ Make sure that you include 'deep-security-agent' in your node's 'run_list'. This
     "recipe[deep-security-agent::default]"
   ]
 }
-```
\ No newline at end of file
+```
+
+## Acknowledgment
+
+* We thank [Blair Hamilton](https://github.com/blairham) for his contributions and put this cookbook into [Supermarket](https://supermarket.chef.io/cookbooks/deep-security-agent).
```

Comment:
change:
'for his contributions and put this cookbook into Supermarket' -> 'for his contributions to the [Deep Security Agent cookbook] (https://supermarket.chef.io/cookbooks/deep-security-agent).'